statefun-e2e-tests/statefun-e2e-tests-common/src/main/java/org/apache/flink/statefun/e2e/common/StatefulFunctionsAppContainers.java

```diff
@@ -142,10 +141,6 @@ public StatefulFunctionsAppContainers(String appName, int numWorkers) {
     this.network = Network.newNetwork();
     this.appName = appName;
     this.numWorkers = numWorkers;
-
-    if (numWorkers > 1) {
-      dynamicProperties.set(CoreOptions.DEFAULT_PARALLELISM, numWorkers);
```

Comment:
Do we actually know that the test succeed because of the -p
This seems to also set the option in the configurations?

Reply:
I verified that the `-p` option is respected, due to the test logs:
```
14:24:17,713 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:17,745 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - Source: org.apache.flink.e2e.sanity-commands-ingress -> router (commands) (1/2) (629795504f9adeabfcdf7bd54a19b208) switched from DEPLOYING to RUNNING.
14:24:17,737 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:17,769 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - Source: org.apache.flink.e2e.sanity-commands-ingress -> router (commands) (2/2) (39160ca2105d2a831cb436043ed42667) switched from DEPLOYING to RUNNING.
14:24:17,953 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:17,985 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - feedback (1/2) (2ac2575233c0b11d009dec544c6dd8e3) switched from DEPLOYING to RUNNING.
14:24:18,070 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:18,103 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - feedback (2/2) (44b80a1071b85a4ea7b2647196199276) switched from DEPLOYING to RUNNING.
14:24:18,071 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:18,104 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - feedback-union -> functions -> Sink: org.apache.flink.e2e.sanity-state-snapshots-egress (2/2) (4103da85baa83f85ab864cf6756fb801) switched from DEPLOYING to RUNNING.
14:24:18,109 INFO  org.apache.flink.statefun.e2e.sanity.SanityVerificationE2E    - 2020-03-14 06:24:18,142 INFO  org.apache.flink.runtime.executiongraph.ExecutionGraph        - feedback-union -> functions -> Sink: org.apache.flink.e2e.sanity-state-snapshots-egress (1/2) (97c8e328a747f0cd071ee468eb4a9a3b) switched from DEPLOYING to RUNNING.
```

As you can see, each operator has 2 subtasks being run.

Reply:
What I mean is, how do you know that this is thanks to the -p and not thanks to the line highlighted above?

Reply:
Ok, scratch that, I’ve looked at the wrong commit.
These lines are actually removed.